replay-scripts/gen-event-names.ts

```diff
@@ -0,0 +1,464 @@
+import fs from 'fs';
```

Comment:
This script needs a comment at the top describing what it is doing.

Reply:
Woops! And it needs a copyright+license header. Coming right up!